src/main/java/net/sf/jabref/logic/integrity/IntegrityCheck.java

```diff
@@ -10,7 +11,11 @@
 import net.sf.jabref.model.entry.ParsedFileField;
 
 import java.io.File;
-import java.util.*;
```

Comment:
OK! I didn't follow it that much, but got the impression that those were the correct settings (I use another version btw, so slightly different look).